src/ext/framework/command.rs

```diff
@@ -14,6 +14,12 @@ pub type AfterHook = Fn(&Context, &Message, &String, Result<(), String>) + Send
 pub type InternalCommand = Arc<Command>;
 pub type PrefixCheck = Fn(&Context) -> Option<String> + Send + Sync + 'static;
 
+#[doc(hidden)]
+pub enum CommandKind {
+    Alias(String),
+    CommandStruct(InternalCommand)
```

Comment:
Just a `Command(InternalCommand)` variant should be fine, appending Struct sounds like a Java-ism